code/game/gamemodes/cult/items/tome.dm

```diff
@@ -8,6 +8,46 @@
 	w_class = 2.0
 	unique = TRUE
 	slot_flags = SLOT_BELT
+	var/obj/item/pen/crayon/chalk/inserted_item = new /obj/item/pen/crayon/chalk/red/cult
+	var/bloody = ""
+
+//chalk code graciously stolen and edited from the PDA's pen code   -MalMalmulam
+/obj/item/pen/crayon/chalk/red/cult
+	name = "engraved chalk"
+	desc = "A piece of chalk for marking areas of floor, or for drawing.  This one has strange symbols engraved on it."
+	color = COLOR_HUMAN_BLOOD	
+ 
+obj/item/book/tome/proc/remove_chalk(mob/user)
+
+	if(use_check_and_message(user))
+		return
+	
+	user.put_in_hands(inserted_item)
+	inserted_item = null
+
+/obj/item/book/tome/verb/verb_remove_chalk()
+	set category = "Object"
+	set name = "Remove chalk"
+	set src in usr
+
+	remove_chalk(usr)
+	
+obj/item/book/tome/attackby(obj/item/C, mob/user)
+	if(istype(C, /obj/item/pen/crayon/chalk)) //Checks if there is a piece of chalk
+		if(inserted_item)
+			to_chat(user, SPAN_NOTICE("There is already \a [inserted_item] in \the [src]."))
+		else
+			user.drop_from_inventory(C,src)
```

Comment:
ideally what you do here is forcemove C into src

Reply:
Hmm, I see that drop_from_inventory runs forcemove twice, once to take it off the mob, once to put it into the destination?  I'm not sure what level of refactoring I'd have to do to make sure substituting forcemove in directly I would need to do, and I'm about to pass out, so I'll that to later.

Reply:
Alright, I've put it in there.  Had to keep drop_from_inventory because it wouldn't properly remove itself from the mob otherwise.